### PR TITLE
Issue #1671: The pedigree editor fails to load when using PhenoTips behind a reverse proxy via mod_proxy_http

### PR DESCRIPTION
**Pull request by veronikaslc**
Fixed. 
@allasm, @danielpgross - can you guys make a quick glance over little change in templateSelector.js?

### CODE REVIEW

**Comment by danielpgross** [comment]
Are the ```pathname``` and ```origin``` properties available in IE9 and up? I looked [here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement#Browser_compatibility), but the answer isn't clear.

**Comment by allasm** [comment]
Looks like <a href="https://developer.mozilla.org/en-US/docs/Web/API/URLUtils/origin">origin</a> is not supported, but there seem to be a <a href="https://github.com/m7r/grunt-ngdocs/issues/106">workaround</a>

**Comment by sdumitriu** [comment]
There's no need for origin, just remove the hostname completely and make path-only request, the browser will know to fill in the right protocol, host and port.

**Comment by sdumitriu** [comment]
I don't think that using the `pathname` attribute is right, for two reasons:

- it's not clear which browsers (and which versions) actually implement it
- it discards the query string, if there is any

File: components/pedigree/resources/src/main/resources/pedigree/view/templateSelector.js [hunk 1]
```diff
@@ -45,7 +45,9 @@ var TemplateSelector = Class.create( {
             pictureBox.update("Loading...");
             this.mainDiv.insert(pictureBox);
             var href = getSelectorFromXML(objects[i], "link", "rel", "http://www.xwiki.org/rel/properties").getAttribute("href");
-            new Ajax.Request(href, {
+            //should use only path when running PhenoTips behind a reverse proxy via mod_proxy_http
+            var path = href.substring(href.indexOf("/rest"));
```
Comment (sdumitriu) [inline components/pedigree/resources/src/main/resources/pedigree/view/templateSelector.js]: This discards the appname from the path. You should substring from the first `/` after `//`:

    var path = href.substring(href.indexOf("/", href.indexOf("//") + 2);

**Pull request merged**